build/configs/qemu/tash_16m/Make.defs

```diff
@@ -137,5 +137,5 @@ HOSTCFLAGS = -Wall -Wstrict-prototypes -Wshadow -g -pipe
 HOSTLDFLAGS =
 
 define DOWNLOAD
-	@gnome-terminal -e "qemu-system-arm -M lm3s6965evb -kernel $(TOPDIR)/../build/output/bin/tinyara -nographic"
+	@gnome-terminal -e "qemu-system-arm -M lm3s6965evb -kernel $(TOPDIR)/../build/output/bin/tinyara -nographic -gdb tcp::3333"
```

Comment:
1. "make run" for other boards is executing serial terminal like putty. It is difficult to support "run" because there are too many programs for it.  We had better not support "run".
2. I agree "run" is better than "download". But it can be download on QEMU and it makes command reduced.